k8s-dev/cstor-pool-mgmt/pool-resource-cluster-1.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: openebs.io/v1alpha1
+kind: CStorPool
+metadata:
+  name: pool1-abc
+  node: node-host-label
+  labels:
+#The following label will be used for additional node reference, since we go for UID based selection of pod.
+    "kubernetes.io/hostname": "node-host-label"
+
+#finalizer helps handling deletion of resource
+  finalizers: ["cstorpool.openebs.io/finalizer"]
```

Comment:
can we make use of k8s resources that injects finalizer into every cstorpool that gets created.

Reply:
As discussed, since CAS template creates this deployment yaml and this is not required to be handled by controller.